src/components/qrGenerator.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+const QRCode = require("qrcode.react");
+
+interface BrightEthereumDeepLinkQRProps {
+  ethAddress: string;
+}
+
+const deepLinkPrefix =
+  "brightid://link-verification/http:%2f%2fnode.brightid.org/Ethereum/";
```

Comment:
```suggestion
  "brightid://link-verification/http://node.brightid.org/ethereum/";
```

Reply:
Follow the [DFE spec](https://docs.google.com/document/d/1c-AqcoMeRMespeK797D2NK33RV7dm0nKCCbygNwLBak/edit#heading=h.siig9uwm002c)  It should look like `brightid://link-verification/http:%2f%2fnode.brightid.org/ethereum/<address>/`